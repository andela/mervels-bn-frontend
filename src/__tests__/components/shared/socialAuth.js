import React from 'react';
import { shallow } from 'enzyme';
import SocialAuth from '../../../components/shared/socialAuth';

describe('LoginPAGE', () => {
    test('should render', () => {
      const wrapper = shallow(<SocialAuth />);
      expect(wrapper).toMatchSnapshot();
    });
});