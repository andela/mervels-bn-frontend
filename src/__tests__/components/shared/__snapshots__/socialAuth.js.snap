// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPAGE should render 1`] = `ShallowWrapper {}`;
